src/sonic-config-engine/minigraph.py

```diff
@@ -1771,17 +1771,16 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
         port_default_speed =  port_speeds_default.get(port_name, None)
         port_png_speed = port_speed_png[port_name]
 
+        ports.setdefault(port_name, {})['speed'] = port_speed_png[port_name]
```

Comment:
given 1772 set the port_png_speed, it is better to use it here as well.

Reply:
fixed